database/tables.py

```diff
@@ -146,3 +146,35 @@ def _list_events(self, start: datetime, end: datetime,
         query = Query()
         records = table.search((query.date > start) & (end > query.date))
         return [Event.deserialize(e) for e in records]
+
+
+# Prefix for all configuration tables.
+CONFIG_TABLES_PREFIX = "configs"
+
+
+class ConfigsTable:
+    """Stores the configurations."""
+
+    def __init__(self, db: Database):
+        self.db = db
+
+    def get(self, config_name: str,
+            config_id: str) -> Optional[Dict[str, str]]:
+        """Retrieves a specific config entry."""
+        table = self.db.get_table(f"{CONFIG_TABLES_PREFIX}/{config_name}")
+        query = Query()
+        entries = table.search(query.id == config_id)
+        if len(entries) > 1:
+            raise ValueError(
+                "Inconsistent database: got multiple records for config id %s"
+                % config_id)
+        return len(entries) == 1 and entries[0] or None
```

Comment:
Please put parenthesis around the different statements, right know it can either be 
```
return ( len(entries)==1 and entries[0] ) or None
```
or
```
return len(entries)==1 and ( entries[0] or None )
```